examples/config-api/config_api.out

```diff
@@ -1,43 +1,40 @@
-# Sensitive data such as passwords can be encrypted and accessed securely in code. To encrypt a value, the
-# `ballerina encrypt` command is used. It prompts the user to enter the value and a secret. In this example,
-# `ballerina` is the value and `12345` is the secret.
-ballerina encrypt
-Enter value:
-
-Enter secret:
+# This example reads two properties (i.e., `username` and `password`) from a configuration file and prints the username in the output. 
 
-Re-enter secret to verify:
+# Sensitive data you have in configuration files such as passwords should be encrypted and accessed securely in code. 
+# Therefore, first, execute the command below to encrypt the `password` property.
+ballerina encrypt
 
-Add the following to the configuration file:
-<key>="@encrypted:{njjb2H9u7Bax2FZX/X+UPZ50lR6+ckKmkesE87Yuw/x9wE1yvhcww1ehOc+mjgCc}"}"
+# This prompts to enter the value (i.e., the password) and a secret (i.e., the key).
+# Enter `ballerina` as the value and `12345` as the secret. 
 
-Or provide it as a command line argument:
---<key>=@encrypted:{njjb2H9u7Bax2FZX/X+UPZ50lR6+ckKmkesE87Yuw/x9wE1yvhcww1ehOc+mjgCc}"}
+# Note the key you get in the output. 
 
-# The secret needs to be provided if a configuration contains an encrypted value. The secret can be provided
-# via a `secret.txt` file (which gets deleted immediately after its contents are read) or via the CLI. In this example, the CLI is used.
-ballerina run config_api.bal
-ballerina: enter secret for config value decryption:
-
-# To specify a configuration file explicitly, use the `--b7a.config.file=<path_to_configuration_file>` property.
-# If this property is not set, Ballerina looks for a `ballerina.conf` file in the current directory. The path
-# to the configuration file can be either an absolute or a relative path.
-# To run this example, place the following configurations in a file and provide its path.
+# Next, create a new  `.conf` or `.txt` (e.g., `ballerina.conf`) file in a preferred location 
+# and place the configurations below to define the two `username` and `password` properties.
+#
 # ```
 # [app.auth]
-#  username=jack
-# [app.auth]
-#  password="@encrypted:{njjb2H9u7Bax2FZX/X+UPZ50lR6+ckKmkesE87Yuw/x9wE1yvhcww1ehOc+mjgCc}"}"
+# username=jack
+# password="@encrypted:{njjb2H9u7Bax2FZX/X+UPZ50lR6+ckKmkesE87Yuw/x9wE1yvhcww1ehOc+mjgCc}"}"
 #
 # ```
-ballerina run config_api.bal --b7a.config.file=path/to/conf/file/custom-config-file-name.conf
-ballerina: enter secret for config value decryption:
 
-Authenticating user 'jack'
+# Set the key as the value of the `password` property in the configuration above. This is to decrypt and read the ‘password` value you encrypted above.
```

Comment:
Is this information necessary for the user with regards to the `Config-Api`.

Reply:
This line seems redundant since we've shown how to place the configurations in the file above.

Reply:
Removed.